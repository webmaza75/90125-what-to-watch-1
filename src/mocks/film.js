const film = {
  genre: [`Drama`, `Thriller`],
  title: `Aviator`,
  desc: ``,
  picture: `aviator.jpg`,
  year: 2016,
  src: ``
};

export default film;
