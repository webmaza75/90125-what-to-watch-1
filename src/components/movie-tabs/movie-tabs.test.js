import React from 'react';
import renderer from 'react-test-renderer';

import MovieTabs from './movie-tabs.jsx';
import reviews from '../../mocks/reviews.js';
import film from '../../mocks/film.js';

it(`MovieTabs correctly renders`, () => {
  const tree = renderer
    .create(<MovieTabs
      reviews={reviews}
      activeItem={`Overview`}
      onChange={jest.fn()}
      movie={film}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
