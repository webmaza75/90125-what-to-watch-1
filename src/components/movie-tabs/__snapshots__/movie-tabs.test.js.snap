// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieTabs correctly renders 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item"
      >
        <a
          className="movie-nav__link"
          href="#"
          onClick={[Function]}
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      6,5
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Good
      </span>
      <span
        className="movie-rating__count"
      >
        386834
         ratings
      </span>
    </p>
  </div>
  <div
    className="movie-card__text"
  >
    As Earth is invaded by alien tripod fighting machines, one family fights for survival.
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Steven Spielberg
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        Tom Cruise, Dakota Fanning, Tim Robbins
      </strong>
    </p>
  </div>
</div>
`;
